### PR TITLE
Feature/transfer tonnages

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
PR for allowing entry of transfer tonnages. Pretty much all testing to be done.

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Scheme/Controllers/TransferEvidenceController.cs [hunk 1]
```diff
@@ -112,18 +113,77 @@ public async Task<ActionResult> TransferFrom(TransferEvidenceNotesViewModel mode
...
+            }
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> TransferTonnage(TransferEvidenceTonnageViewModel model)
+        {
+            using (var client = this.apiClient())
+            {
+                if (ModelState.IsValid)
```
Comment (NateHogan) [inline src/EA.Weee.Web/Areas/Scheme/Controllers/TransferEvidenceController.cs:161]: Is this going to be used in the future? Could just be removed

**Review by NateHogan — COMMENTED** [review]
I left two comments, looks good - if not a little complicated :)

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceNotesViewModelMapTransfer.cs [hunk 2]
```diff
@@ -14,6 +15,10 @@ public class TransferEvidenceNotesViewModelMapTransfer
 
         public Guid OrganisationId { get; private set; }
 
+        public TransferEvidenceTonnageViewModel ExistingModel { get; set; }
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceNotesViewModelMapTransfer.cs]: have removed... :)

**Comment by JohnPadgettCivica** [comment]
> I left two comments, looks good - if not a little complicated :)

tell me about it...

**Review by NateHogan — APPROVED** [review]

**Pull request merged**